packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -30,14 +30,18 @@ const propTypes = {
    * Indicates whether the item is expanded. Only items with subfolderItems can be expanded.
    */
   isExpanded: PropTypes.bool,
+  /**
+   * Indicates whether the item can be selected. Selectable items will display with a radio button.
+   */
+  isSelectable: PropTypes.bool,
   /**
    * Indicates whether the item is selected. Because this component has the appearance of a radio button group, only one item should be selected at a time.
    */
   isSelected: PropTypes.bool,
   /**
    * The callback function for a click event.
    */
-  onClick: PropTypes.func,
+  onSelect: PropTypes.func,
```

Comment:
Renamed this prop from `onClick` to `onSelect` to better reflect its actual intended usage. If there are concerns about passivity this can be reverted.

Reply:
I think the passivity concern would be if MPages is already using this callback.  If they have coded to use onClick, this minor version update would break them, correct?

Reply:
Yes, this will break consumers who are already using onClick.

Reply:
The component is still in alpha, so I think we can get away with making a non-passive change. The prop rename makes sense to me. I would just add a "breaking change" entry in the CHANGELOG.

Reply:
Good point, added the breaking change entry here: [7555293](https://github.com/cerner/terra-framework/pull/2021/commits/75552939d325bc7006aca75820c0d60b958639f8)

Reply:
+1